arelle/plugin/validate/HMRC/__init__.py

```diff
@@ -25,6 +25,47 @@
 EMPTYDICT = {}
 _6_APR_2008 = dateTime("2008-04-06", type=DATE)
 
+COMMON_GENERIC_DIMENSIONS = {"Chairman": ("NameEntityOfficer",),
```

Comment:
Optional formatting nit, we can just do this in a subsequent PR: If you move the first dict entry onto a new line, formatting will naturally only use a single indent here rather than pushing it out to line up with the bracket. Same with `"charities": COMMON_GENERIC_DIMENSIONS | ...` below.